README.md

```diff
@@ -181,11 +181,11 @@ Please learn more about translating our products: https://adguard.com/kb/miscell
 ## <a id="templates"></a> Filters templates
 `@include` directive allows to include the content of specified file into the filter.
 
-If you want to add a filter and keep all its rules during optimization, use the `/notOptimized` parameter.
+More information about the `@include` directive and its options can be found here: https://github.com/AdguardTeam/FiltersCompiler/#include-directive.
 
 Example:
 ```
-@include "https://example.org/test-filter.txt" /notOptimized
+@include "https://example.org/test-filter.txt" /notOptimized /addModifiers="script" /exclude="../exclusions.txt" /stripComments
```

Comment:
the example should be removed as well to avoid duplicates of docs

Reply:
Why? Description now in two repos. Info about metadata and directives must be here.

Reply:
to avoid duplicates and have description of `@include` directive only in FiltersCompiler repo. but I don't mind if it's ok to you